hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/metrics/DataNodeDiskMetrics.java

```diff
@@ -127,6 +142,21 @@ public void run() {
 
             detectAndUpdateDiskOutliers(metadataOpStats, readIoStats,
                 writeIoStats);
+
+            // Sort the slow disks by latency and .
```

Comment:
@tomscut Happy New Year! Sorry for taking a long time.
The comment in this line seems incomplete. Could you fix or remove it? Then +1 from me.

Reply:
@tasanuma Happy New Year! I am sorry for this and I will fix it right away.